Rubberduck.Main/ComClientLibrary/UI/DockableWindowHost.cs

```diff
@@ -347,7 +347,13 @@ private void OnCallBackEvent(object sender, SubClassingWindowEventArgs e)
                 return;
             }
             var param = new LParam { Value = (uint)e.LParam };
-            _userControl.Size = new Size(param.LowWord, param.HighWord);
+            // The VBE passes a special value to the HighWord when docking into the VBE Codepane 
+            // instead of docking into the VBE itself.
+            // that special value (0xffef) blows up inside the guts of Window management because it's
+            // apparently converted to a signed short somewhere and then considered "negative"
+            // that is why we drop the signbit for shorts off our values when creating the control Size.
+            const ushort signBitMask = 0x8000;
+            _userControl.Size = new Size(param.LowWord & ~signBitMask, param.HighWord & ~signBitMask);
```

Comment:
Just checking - would it be better to simply ignore the windows message when its signbit is set instead of setting the size to some value that might not be sensible?

Reply:
@bclothier I've emailed Cristy in the VBE team to hopefully find out. I'll merge this PR as-is in the meantime.